### ISSUE TITLE
Corrupted download?

### ISSUE THREAD

**Issue by Kidguru95** [PathOfBuildingCommunity/PathOfBuilding#6356]
### Check version

- [X] I'm running the latest version of Path of Building and I've verified this by checking the [changelog](https://github.com/PathOfBuildingCommunity/PathOfBuilding/blob/master/CHANGELOG.md)

### Check for duplicates

- [X] I've checked for duplicate issues by using the search function of the [issue tracker](https://github.com/PathOfBuildingCommunity/PathOfBuilding/issues)

### How is Path of Building expected to behave?

Normally after downloading it I'm able to use it pretty smoothly especially since I'm a guide andy but this time it doesn't even want to open correctly even after reinstalling multiple times.

### How does Path of Building behave?

It simply opens for me and gives me an error code immediately after trying to update.

### Screenshots

![image](https://github.com/PathOfBuildingCommunity/PathOfBuilding/assets/142346575/58457926-e187-4639-91f7-5f0da7ce7244)

**Comment by Vipera222** [comment]
Got same issue, tried downloading yesterday's version it got same issue too.

**Comment by Kidguru95** [comment]
Thanks for the info at least we can start figuring out that it's more than just me lol.

**Comment by Aluvis** [comment]
Just downloaded PoB and got this exact same issue.

**Comment by Vipera222** [comment]
Just checked and zip works tho, only with exe issue.

**Comment by Wires77** [comment]
This looks like a OneDrive issue to me.  If you go to Documents/Path of Building and open your Settings.xml file manually, does PoB work properly after that?  The file API PoB uses doesn't play nicely with however OneDrive dynamically moves files to the cloud and back.

**Comment by Kidguru95** [comment]
Whenever I try to open the onedrive I just get an error saying it can't reach it and I can't delete it either unless I should open the xml file another way besides the browser option?

**Comment by Kidguru95** [comment]
I'm not seeing that option anywhere I look.

**Comment by Wires77** [comment]
Are you on Windows 11?  The file is under your OneDrive, correct?  If not I'm not sure what the issue could be, then.  As a last resort, if you delete the file, PoB will recreate it upon starting, but you'll lose any settings you had in place.

**Comment by Kidguru95** [comment]
Yea im on windows 11 so I'm not sure where that option is

**Comment by Kidguru95** [comment]
Ok so I had to download one drive before everything started working again even though onedrive sucks D:.

**Comment by Aluvis** [comment]
> Just checked and zip works tho, only with exe issue.

Yep, the zip also worked for me. No idea what's up with the .exe version.
